.github/workflows/CI.yml

```diff
@@ -14,16 +14,17 @@ env:
    REDIS_HOST: localhost
    REG_VERSION: v2.7.1-patch-2819-2553
    UI_BUILDER_VERSION: 1.6.0
+   CODECOV_TOKEN: 89c7b95b-dbce-431e-8cb3-55691a9dcc64 #codecov token: visit https://github.com/codecov/codecov-action to learn more
```

Comment:
It's a plaintext secret

Reply:
because I can not think of a better solution

Reply:
@AllForNothing 

Have you tried other possible ways?

Reply:
https://github.community/t5/GitHub-Actions/Make-secrets-available-to-builds-of-forks/m-p/30685#M511    I found the answer here.  It indicate there is no other way so far

Reply:
it seems the only risk is hacker may post a fake test coverage report with this token